### ISSUE TITLE
[eslint-plugin-lit-a11y] scope should be allowed on custom elements

### ISSUE THREAD

**Issue by thepassle** [open-wc/open-wc#1886]
Currently failing test, this should pass:

```html
<foo-bar scope="foo"></foo-bar>
```

We can just add a check to see if the `element.name.includes('-')` . If it does include a dash, it means it's a custom-element, and we should allow it to have a `scope` attribute.

https://github.com/open-wc/open-wc/blob/d96c307c2f4c236425b702b2db6be1b6a152549b/packages/eslint-plugin-lit-a11y/lib/rules/scope.js#L34
